src/MarginTrading.SettingsService.SqlRepositories/Repositories/TradingInstrumentsRepository.cs

```diff
@@ -228,5 +230,61 @@ public async Task DeleteAsync(string assetPairId, string tradingConditionId)
 
             return objectsToAdd;
         }
+
+        public async Task<List<ITradingInstrument>> UpdateBatchAsync(string tradingConditionId,
+            List<TradingInstrument> items)
+        {
+            var entities = items.Select(_convertService.Convert<TradingInstrument, TradingInstrumentEntity>).ToList();
+            
+            using (var conn = new SqlConnection(_connectionString))
+            {
+                SqlTransaction transaction = null;
+                
+                try
+                {
+                    if (conn.State != ConnectionState.Open)
+                    {
+                        await conn.OpenAsync();
+                    }
+                    
+                    transaction = conn.BeginTransaction();
+
+                    var ids = string.Join(",", entities.Select(x => $"'{x.TradingConditionId}{x.Instrument}'"));
+                    var idsCond = $"TradingConditionId + Instrument IN ({ids})";
```

Comment:
Why... You can use 
`WHERE TradingConditionId = @tradingConditionId AND Instrument IN @instruments`
and inject parameters